packages/binding-http/src/http-server.ts

```diff
@@ -424,7 +422,7 @@ export default class HttpServer implements ProtocolServer {
             }
             case "Digest":
                 return false;
-            case "OAuth":
+            case "OAuth": {
```

Comment:
Similar, this seems a matter of taste, right? No need to encapsulate..

Reply:
eslint recommends this in the default recommanded rules